lib/govuk_publishing_components/presenters/machine_readable/dataset_schema.rb

```diff
@@ -29,6 +29,10 @@ def name
           "name" => page.title
         }
       end
+
+      def truncate_description(string)
+        string.length > 5000 ? string.slice(0..4999) : string
```

Comment:
`slice` doesn't extent the list if it's shorter than your ranger, so this can be `string.slice(0..4999)`.  or the whole thing could even be `(page.body || page.description).slice(0..4999)`

Reply:
done, got rid of the extra method and sliced the original description method